scripts/release/setup-workspace

```diff
@@ -121,11 +96,8 @@ else
 fi
 
 scriptdir="$(dirname "$0")"
-if [ $extra_disabled -eq 1 ]; then
-    "$scriptdir/mel-checkout" -X "$WORKSPACEDIR" "$manifest"
-else
-    "$scriptdir/mel-checkout" "$WORKSPACEDIR" "$manifest" $extra_manifests
-fi && cd "$WORKSPACEDIR"
+"$scriptdir/mel-checkout" ${extra_disabled+-X} "$WORKSPACEDIR" "$manifest" $extra_manifests \
```

Comment:
@kergoth, it looks like the -x option is not behaving as it should and although extra manifests are being passed to mel-checkout the -X option always shows up and borks the execution.

Reply:
Hmm, interesting. ${foo+bar} expands to bar only when foo is set, generally. I'll take a closer look, thanks for testing. I did do basic build testing with this and it worked fine there, but I must have missed something.